### DISCUSSION TITLE
Adding a custom logo

### DISCUSSION THREAD

**Discussion by flayks** [sanity-io/sanity#3733] [Sanity Studio Support] [answered]
Hey!
I saw the new update for preview-21 and tried to customize the Studio logo but this doesn't seem to work 🤔
Am I missing something?

```js
// sanity.config.js
import Logo from './custom/Logo'
export default createConfig({
  // ...
  studio: {
    components: {
      logo: Logo,
    },
  },
  // ...
})
```

```jsx
// Logo.jsx
import React from 'react'

export default function LogoButton () {
    return (
        some <svg>
    )
}
```

**Comment by geball** [comment, marked as answer]
@flayks `dev-preview` was [rolled back](https://github.com/sanity-io/sanity/discussions/3729#discussioncomment-3816683) to `preview.20` so you may not actually be on `preview.21`.

**Reply by flayks** [reply]
That is… correct! Did not see that haha.
Indeed the `logo:` property in `createConfig` is still here (but it works!). Thanks for the investigation 🕵️
